deploy-web/src/components/deployment/DeploymentListRow.tsx

```diff
@@ -110,7 +110,7 @@ export const DeploymentListRow: React.FunctionComponent<Props> = ({ deployment,
   const hasActiveLeases = hasLeases && filteredLeases.some(l => l.state === "active");
   const deploymentCost = hasLeases ? filteredLeases.reduce((prev, current) => prev + parseFloat(current.price.amount), 0) : 0;
   const timeLeft = getTimeLeft(deploymentCost, deployment.escrowBalance);
-  const realTimeLeft = useRealTimeLeft(deploymentCost, deployment.escrowBalance, deployment.escrowAccount.settled_at, deployment.createdAt);
+  const realTimeLeft = useRealTimeLeft(deploymentCost, deployment.escrowBalance, parseFloat(deployment.escrowAccount.settled_at), deployment.createdAt);
```

Comment:
I'm pretty sure it's always a string, it wasn't breaking because in `useRealTimeLeft` we do `settledAt - latestBlockHeight` and by doing a subtraction it gets converted into a number

Reply:
Ah I see makes sense